### ISSUE TITLE
1.15.0 can't find task w/ name "check"

### ISSUE THREAD

**Issue by kenyee** [jeremymailen/kotlinter-gradle#65]
plugins {
    id 'org.jmailen.kotlinter' version '1.15.0'
}
in the root gradle (not the app gradle) file causes this error
```
* What went wrong:
An exception occurred applying plugin request [id: 'org.jmailen.kotlinter', version: '1.15.0']
> Failed to apply plugin [id 'org.jmailen.kotlinter']
   > Task with name 'check' not found in root project 'android'.
```

1.14.0 didn't have a problem.
We do have flavors and build types.
Switching back to 1.14.0, gradle tasks does show the "check" task:

```
Verification tasks
------------------
check - Runs all checks.
connectedAndroidTest - Installs and runs instrumentation tests for all flavors on connected devices.
```

**Comment by jeremymailen** [comment]
Thanks for the report and sorry for the trouble. I'll look into it ASAP.

**Comment by jeremymailen** [comment]
Failure fixed in `1.15.1` and documentation added: https://github.com/jeremymailen/kotlinter-gradle#multi-module-and-android

**Comment by kenyee** [comment]
"It does look like I've accidentally removed the safeguards that prevent the creation of the parent tasks from blowing up if there is no check (or source code) in the current module"

That explains why I never saw any warnings from it but Detekt flagged stuff 😂
